operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/ConfigurationService.java

```diff
@@ -42,4 +42,17 @@ default Config getClientConfiguration() {
    * @return the version information
    */
   Version getVersion();
+
+  /**
+   * Whether the operator should query the CRD to make sure it's deployed and validate {@link
+   * CustomResource} implementations before attempting to register the associated controllers.
+   *
+   * <p>Note that this might require elevating the privileges associated with the operator to gain
+   * read access on the CRD resources.
+   *
+   * @return {@code true} if CRDs should be checked (default), {@code false} otherwise
+   */
+  default boolean validateCustomResources() {
```

Comment:
It should be validateCustomResourceDefinitions

Reply:
Not really, we're validating the Custom Resources from the CRD: if the CRD has been accepted by the cluster, it's presumed to be valid. However, the `CustomResource` implementation you want to use in your controller might not match what the CRD defines so it's really validating that these classes match what the cluster knows and wants.

Reply:
ah ok, got it! then ignore my other comments too and go ahead to merge the PR :)